### DISCUSSION TITLE
新功能询问 - 游玩时间

### DISCUSSION THREAD

**Discussion by UWuxi** [Meloong-Git/PCL#4397] [提问] [answered]
请问，可以加一个关于账号游玩时间的统计吗？玩的整合包有点多，想知道可不可以在登录账号后有一个记录全部存档游玩时间的显示？这个功能虽说有点鸡肋，但我在写游戏策划简历的时候，遇到了想写Minecraft作为游戏经历却无法估计自己游玩时间的情况。

**Comment by allMagicNB** [comment, marked as answer]
#1717
